src/pages/HomePage.jsx

```diff
@@ -28,14 +37,18 @@ export default function HomePage() {
               <p className="mt-4 text-justify">
                 Join us today and discover the joy of fitness!
               </p>
-            </div>
+            </motion.div>
           </div>
         </div>
       </section>
       <section>
         <div className="mx-auto max-w-screen-2xl px-4 py-8 sm:px-6 lg:px-8">
           <div className="grid grid-cols-1 gap-4 md:grid-cols-2">
-            <div className="bg-blue-900 p-8 md:p-12 lg:px-16 lg:py-24">
+            <motion.div 
+            // initial={{ opacity: 0, y: 200 }}
+            // animate={{ opacity: 1, y: 0 }}
+            // transition={{ duration: 2 }}
```

Comment:
@Lalman888 did you want to add an animation here as well?

Reply:
I wanted to add animation, but don't look good any suggestions ?

Reply:
maybe a scroll-triggered animation that starts when the element is shown? could defining only opacity

Reply:
okay I will do that

Reply:
here https://deploy-preview-146--gymjunkies.netlify.app/        I didn't see any problem 
 
![image](https://github.com/gabrysia694/Gym-Junkies/assets/90516185/da903dd0-ccb9-4fae-a49c-132c57449a87)

Reply:
I meant the horizontal scrollbar is showing up for the time of animation and we could hide it so it looks better

Reply:
Can you merge this then again I will open new pull request for overflow-x-hidden this creating merge conflict .